packages/frontend-plugin-api/src/wiring/resolveExtensionDefinition.ts

```diff
@@ -126,6 +137,49 @@ export type ResolveExtensionId<
     : never
   : never;
 
+function resolveExtensionId(
+  kind?: string,
+  namespace?: string,
+  name?: string,
+): string {
+  const namePart =
+    name && namespace ? `${namespace}/${name}` : namespace || name;
```

Comment:
good thing we know all operator precedence by heart and never get confused by stuff like this right

(i know prettier helpfully does this :) )

Reply:
yup ._.